samples/excel/90-scenarios/performance-optimization.yaml

```
@@ -11,75 +11,132 @@ script:
         $("#refresh-data").click(() => tryCatch(refreshData));
         $("#toggle-tracking").click(() => tryCatch(toggleTracking));
         $("#toggle-screen-painting").click(() => tryCatch(toggleScreenPainting));
+        $("#toggle-calculation").click(() => tryCatch(toggleCalculationMode));
+        $("#recalculate").click(() => tryCatch(recalculate));
+
+        const ROW_COUNT = 500;
+        const COLUMN_COUNT = 20;
 
         let untrack = false;
         let pauseScreenPainting = false;
 
         async function refreshData() {
-            await Excel.run(async (context) => {
-                const sheet = context.workbook.worksheets.getActiveWorksheet();
-                const rowCount = 500;
-                const columnCount = 20;
-
-                const startTime = Date.now();
-                console.log("Starting...");
-
-                if (pauseScreenPainting) {
-                    // Note: Don't call `suspendScreenUpdatingUntilNextSync` repeatedly (such as in a loop).
-                    // Repeated calls will cause the Excel window to flicker. 
-                    context.application.suspendScreenUpdatingUntilNextSync();
-                }
+          await Excel.run(async (context) => {
+            // Recreate the data in the worksheet with random data.
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
 
-                for (let i = 0; i < rowCount; i++) {
-                    for (let j = 0; j < columnCount; j++) {
-                        let cell = sheet.getCell(i, j);
-                        cell.values = [[i * j * Math.random()]];
-                        if (untrack) {
-                            cell.untrack();
-                        }
-                    }
+            const startTime = Date.now();
+            console.log("Starting...");
+
+            // If other parts of the sample have toggled screen printing off, this will stop screen updating until context.sync is called.
```

Should this be screen painting instead of screen printing? 
```suggestion
            // If other parts of the sample have toggled screen painting off, this will stop screen updating until context.sync is called.
```